### DISCUSSION TITLE
Does Silo keep non-processed calls for later?

### DISCUSSION THREAD

**Discussion by jmairena-s3d** [dotnet/orleans#7090] [Q&A]
Hello!

I'm doing a stress test on a Silo for 10 minutes.
While running it my CPU is neraly at 100% all the time, but I get no critical messages on the Silo output, just some operations that have taken more than expected, but not too much.

After 10 minutes of stress test, I close everything but the Silo. 
CPU usage back to 5% and Silo received requests to 0.
But after 1 minute of calm, Silo receives a huge wave of requests:

![imagen](https://user-images.githubusercontent.com/84007669/119620469-4ebc2880-be05-11eb-828f-e3f1138b3a16.png)

I don't know where these requests are coming from since I have closed everithing but the Silo.
Does the Silo save non-processed calls for later maybe?

**Comment by ReubenBond** [comment]
Is this a test with one silo and several clients? Is the stress test simple request-response from client to silo or are there other messaging patterns involved?

**Reply by jmairena-s3d** [reply]
1 Silo and up to 1000 clients at the same time.

Clients are doing more than just 1 call, each one has a bunch of different calls. 
But those calls are done sequentially, so no call is done by the client before receiving the response of the previous call.
